### ISSUE TITLE
the google assistance is blocked on badblock+

### ISSUE THREAD

**Issue by AyanokojiK1yotaka** [celenityy/bugs#25]
assistant-s3-pa.googleapis.com

**Comment by celenityy** [comment]
Thanks for the report, but that's expected, we block Google Assistant due to the highly invasive nature of it. I might consider organizing/laying this out better though. (For instance, maybe not blocking ex. Google Assistant in the lower lists).

That's I guess one of the goals of this project, we're fine blocking invasive functionality like that *if its justified*. We want to avoid breakage where possible, don't get me wrong, but for something like Google Assistant, yeah, that's *extremely* problematic from a privacy perspective, and I don't think I'm comfortable allowing that. 

I do carefully document domains though on the lists themselves, since I also want to make it as easy as possible for people to whitelist domains that they need to. Feel free to keep this one on your whitelist. I think that's a benefit of blocking this & how BadBlock is designed; it forces the user to think about this & make an active decision whether they want to use questionable functionality like Google Assistant or not, rather than it just being left as the default (or turned on through dark patterns) and automatically made for them. 

Please let me know if you have any other issues or concerns, always glad to help, and appreciate your support and checking out the project, especially when getting started like this.

**Comment by celenityy** [comment]
Migrated to our new issue tracker at https://codeberg.org/celenity/bugs/issues/25, hope to see you there!
